Tests/Marketplace/Tests/marketplace_services_test.py

```diff
@@ -937,3 +1069,119 @@ def test_set_dependencies_core_pack_mandatory_dependency_override(self):
+            }
+        }
+        mocker.patch('builtins.open', mock_open(read_data=original_changelog))
+        changelog, changelog_latest_rn_version = dummy_pack.get_changelog_latest_rn('fake_path')
+        assert changelog == original_changelog_dict
+        assert changelog_latest_rn_version == LooseVersion('2.0.0')
+
+    def test_create_local_changelog(self, mocker, dummy_pack):
+        """
+           Given:
+               - Changelog file path of the given pack withing the index dir
+           When:
+               - Creating the local changelog under the pack path
+           Then:
+               - Verify that the local changelog file has been created successfully
+       """
+        mocker.patch('os.path.exists', return_value=True)
+        mocker.patch('shutil.copyfile')
+        mocker.patch('os.path.isfile', return_value=True)
+        mocker.patch("Tests.Marketplace.marketplace_services.logging")
+        task_status = dummy_pack.create_local_changelog('fake_path')
+        assert task_status
+
+    def test_get_release_notes_lines_aggregate(self, mocker, dummy_pack):
+        """
+           Given:
+               - 3 release notes files, 1.0.0 is the latest rn, 1.0.1 and 1.0.2 are new rn files
+           When:
+               - Creating the release notes for the new version (1.0.2)
+           Then:
+               - Verify that the rn of 1.0.1 and 1.0.2 are aggregated
+       """
+        rn_one = '''
+#### Integrations
+##### CrowdStrike Falcon Intel v2
+- wow1
+        '''
+        rn_two = '''
+#### Integrations
+##### CrowdStrike Falcon Intel v2
+- wow2
+        '''
+        aggregated_rn = "\n#### Integrations\n##### CrowdStrike Falcon Intel v2\n- wow1\n- wow2\n"
+        open_mocker = MockOpen()
+        mocker.patch('os.listdir', return_value=['1_0_0.md', '1_0_1.md', '1_0_2.md'])
+        open_mocker['rn_dir_fake_path/1_0_1.md'].read_data = rn_one
+        open_mocker['rn_dir_fake_path/1_0_2.md'].read_data = rn_two
+        mocker.patch('builtins.open', open_mocker)
+        rn_lines, latest_rn = dummy_pack.get_release_notes_lines('rn_dir_fake_path', LooseVersion('1.0.0'))
+        assert latest_rn == '1.0.2'
+        assert rn_lines == aggregated_rn
+
+    def test_get_release_notes_lines_updated_rn(self, mocker, dummy_pack):
+        """
+           Given:
+               - 2 release notes files, 1.0.1 is the latest rn and exists in the changelog
+           When:
+               - Creating the release notes for version 1.0.1
+           Then:
+               - Verify that the rn are the same
+       """
+        rn = '''
+#### Integrations
+##### CrowdStrike Falcon Intel v2
+- wow1
+        '''
+        mocker.patch('builtins.open', mock_open(read_data=rn))
+        mocker.patch('os.listdir', return_value=['1_0_0.md', '1_0_1.md'])
+        rn_lines, latest_rn = dummy_pack.get_release_notes_lines('rn_dir_fake_path', LooseVersion('1.0.1'))
+        assert latest_rn == '1.0.1'
+        assert rn_lines == rn
```

Comment:
yes, take a look at the if-else clause in `Pack.get_release_notes_lines`, the `else` case in line `986` handles two scenarios:
1. In case where there is only one new release notes file, OR
2. In case where the pack is up to date, i.e. latest changelog is latest rn file
btw, there's a comment in the code with the above scenarios.

Reply:
Ok so let's add a case for when the latest version in the directory is the same as the changelog